src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -14,36 +13,27 @@
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 
-
 @Component
-public class DataLoader
-{
-
+public class DataLoader {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUndocumentedApi)